src/components/Modals/LedgerOpenApp/hooks/useLedgerAppDetails.tsx

```diff
@@ -0,0 +1,21 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { ethAssetId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import { useMemo } from 'react'
+import { selectAssetById, selectFeeAssetByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export const useLedgerAppDetails = (chainId: ChainId) => {
+  const asset = useAppSelector(state => selectFeeAssetByChainId(state, chainId))
```

Comment:
```suggestion
  const feeAsset = useAppSelector(state => selectFeeAssetByChainId(state, chainId))
```

Reply:
Self-addressed in https://github.com/shapeshift/web/pull/7386/commits/ac7f3c2898a2f002a228baf798a839df4068076b